ttnn/cpp/ttnn/operations/experimental/transformer/device/transformer_op.hpp

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <optional>
+
+#include "tensor/tensor.hpp"
+#include "tt_dnn/op_library/run_operation.hpp"
+
+namespace ttnn::operations::experimental::transformer {
+
+constexpr uint8_t DefaultQueueId = 0;
+
+struct ConcatenateHeads {
```

Comment:
rename this to `ConcatenateHeadsDeviceOperation` please

Reply:
Done